CONTRIBUTING.md

```diff
@@ -134,8 +134,10 @@ If you are making changes to the documentation at [https://llama-stack.readthedo
 $ cd llama-stack/docs
 $ uv sync --extra docs
 
+# This rebuilds the documentation pages.
+$ uv run make html
+
 # This will start a local server (usually at http://127.0.0.1:8000) that automatically rebuilds and refreshes when you make changes to the documentation.
-$ make html
 $ uv run sphinx-autobuild source build/html
```

Comment:
can you also add a `--write-all` here otherwise autobuild keeps older stale things around

Reply:
Updated!